brozzler/metrics.py

```diff
@@ -0,0 +1,59 @@
+from typing import Optional
+
+try:
+    from http_sd_registry.client import (
+        Client,
+        Env,
+        Registration,
+        Scheme,
+        format_self_target,
+    )
+    from http_sd_registry.config import ClientConfig
+except ImportError:
+    http_sd_registry = None
+
+
+from prometheus_client import Counter, Gauge, Histogram, start_http_server
+
+# fmt: off
+brozzler_pages_crawled = Counter("brozzler_pages_crawled", "number of pages visited by brozzler")
+brozzler_page_processing_duration_seconds = Histogram("brozzler_page_processing_duration_seconds", "time spent processing a page in brozzler")
```

Comment:
The default Histogram buckets are likely not appropriate for this use case; the maximum bucket is `10`: https://github.com/prometheus/client_python/blob/master/prometheus_client/metrics.py#L586